apache/superset
# fix: Added chart examples for left out chart types

Added chart examples for left out chart types and modified VizTypeGallery.tsx for rendering thumbnail image in case of no example provided


### SUMMARY

Adding chart examples for Deck.gl:
* Arc
* Grid
* Hexagon
* Path
* Polygon
* Scatterplot
* Screen Grid
* Contour
* Heatmap
* Geojson

And made a change in VizTypeGallery.tsx for rendering thumbnail image in case of no example provided

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

#### BEFORE
![image](https://github.com/apache/superset/assets/95441117/d0c2b82a-e97c-4eef-92ae-90835566b28c)

#### AFTER
[![After Changes]](https://github.com/apache/superset/assets/95441117/a79421ec-d488-4c34-a069-273e82117c09)

### TESTING INSTRUCTIONS
Kindly go to [http://localhost:9000/chart/add](http://localhost:9000/chart/add) and please select any chart so that you can be able to see example images of selected chart

### ADDITIONAL INFORMATION
